example/tutorial/Hierarchical_Parallelism/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 
-# FIXME_SYCL requires TeamPolicy
-IF(NOT KOKKOS_ENABLE_SYCL)
 KOKKOS_ADD_EXAMPLE_DIRECTORIES(01_thread_teams)
 KOKKOS_ADD_EXAMPLE_DIRECTORIES(01_thread_teams_lambda)
 KOKKOS_ADD_EXAMPLE_DIRECTORIES(02_nested_parallel_for)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(03_vectorization)
+# FIXME_SYCL needs Random
+IF(NOT Kokkos_ENABLE_SYCL)
```

Comment:
```suggestion
IF(NOT KOKKOS_ENABLE_SYCL)
```

Reply:
This rather belongs to #3818. Let me fix it there and then rebase here (or I drop the related commits here).

Reply:
Fixed in 4a1f2675cb663de9a2fbdf84195e3a998cd9f055.

Reply:
...and rebased.